resource_retriever/test/CMakeLists.txt

```diff
@@ -1,6 +1,5 @@
-set(EXECUTABLE_OUTPUT_PATH ${PROJECT_SOURCE_DIR})
+ament_add_gtest(${PROJECT_NAME}_test test.cpp)
 
-catkin_add_gtest(${PROJECT_NAME}_utest test.cpp)
-target_link_libraries(${PROJECT_NAME}_utest ${PROJECT_NAME})
-
-catkin_add_nosetests(test.py)
+if(TARGET ${PROJECT_NAME}_test)
+  target_link_libraries(${PROJECT_NAME}_test ${PROJECT_NAME})
+endif()
```

Comment:
The top-level CMakeLists.txt has no `include` or `add_subdirectory` directives that I can see.  How is this getting called and built?

Reply:
I forgot to remove it https://github.com/ros/resource_retriever/pull/43/commits/edd0621053d8eb60d5145d1caab15542a7bd583a 

I moved all the logic to the CMakeLists.txt like ROS2 packages